packages/gatsby-plugin-layout/src/wrap-page.js

```diff
@@ -0,0 +1,9 @@
+const React = require(`react`)
+
+const preferDefault = m => (m && m.default) || m
+const Layout = preferDefault(require(GATSBY_LAYOUT_COMPONENT_PATH).default)
```

Comment:
Thanks. woopsie on my part by not removing `.default` - and we need to be flexible because it's up to user to either es6 export default or module export and we need to handle both